docs/src/index.md

```diff
@@ -4,7 +4,33 @@ CurrentModule = Taxonomy
 
 # Taxonomy
 
-Documentation for [Taxonomy](https://github.com/formal-methods-mpi/Taxonomy.jl).
+## What?
+
+Taxonomy.jl aims to serve as a comprehensive database of structural equation models (SEMs) that can be used to infer distributions of both structures (e.g., types of models, numbers of observed and latent variables) and parameters (e.g., what is the average factor loading).
+This will greatly facilitate simulation studies that accurately reflect real-world conditions and takes seriously the idea that "Simulation studies are to a statistician what experiments are to a scientist" [Pawel & Kook et al.](https://doi.org/10.1002/bimj.202200091).
```

Comment:
```suggestion
This will greatly facilitate simulation studies that accurately reflect real-world conditions and take the idea seriously that "Simulation studies are to a statistician what experiments are to a scientist" [Pawel & Kook et al.](https://doi.org/10.1002/bimj.202200091).
```